packages/manager/modules/pci/src/projects/project/storages/databases/database/users/users.routing.js

```diff
@@ -1,145 +1,60 @@
 import map from 'lodash/map';
 import User from '../../../../../../components/project/storages/databases/user.class';
-import isFeatureActivated from '../../features.constants';
 import { SECRET_TYPE } from '../../databases.constants';
+import { STATUS } from '../../../../../../components/project/storages/databases/databases.constants';
 
 export default /* @ngInject */ ($stateProvider) => {
   $stateProvider.state(
     'pci.projects.project.storages.databases.dashboard.users',
     {
       url: '/users',
       views: {
-        databaseView: 'pciProjectsProjectUsers',
+        databaseView: 'ovhManagerPciStoragesDatabaseUsersComponent',
       },
       resolve: {
         breadcrumb: /* @ngInject */ ($translate) =>
-          $translate.instant('pci_projects_project_users_title'),
-        users: /* @ngInject */ (DatabaseService, database, projectId) =>
-          DatabaseService.getUsers(
+          $translate.instant('pci_databases_users_title'),
+        users: /* @ngInject */ (DatabaseService, database, projectId) => {
+          // Poll the database status
+          if (database.isProcessing()) {
+            DatabaseService.pollDatabaseStatus(
```

Comment:
**question:** Does the Polling is killed once user move to another state? (Used to be done on `$onDestroy` hook).

Reply:
Good catch, I forgot to stop the polling, I've added it here: https://github.com/ovh/manager/pull/6090/commits/ea60c8f86c638103f84054aff088896cc6192f2b